docs/appengine/cloudsql_postgres/cloudsql.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample cloudsql demonstrates connection to a Postgres Cloud SQL instance from App Engine standard.
+package cloudsql
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	_ "github.com/broady/gae-postgres"
+)
+
+func init() {
+	http.HandleFunc("/", handler)
+}
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+
+	connectionName := mustGetenv("CLOUDSQL_CONNECTION_NAME")
+	user := mustGetenv("CLOUDSQL_USER")
+	password := os.Getenv("CLOUDSQL_PASSWORD") // NOTE: password may be empty
+
+	w.Header().Set("Content-Type", "text/plain")
+
+	db, err := sql.Open("gae-postgres", fmt.Sprintf("user=%s password=%s cloudsql=%s", user, password, connectionName))
```

Comment:
@broady I wonder if we should instead open this outside of the handler so that the sample won't have to open a new connection each time. However, if making a global DB connection is undesirable, then this is works.

Reply:
Huh! Weird. When I wrote this sample (a long time ago), I assumed you'd need to pass in a ctx somewhere.

It seems obvious now that GAE would let you connect to the database without the context of an incoming request. Cool.

Will leave this PR for now, and update the MySQL sample tomorrow first.

Reply:
Awesome, cool. Catch you later!